src/core/computeDeltaU.m

```diff
@@ -16,16 +16,20 @@
 % You should have received a copy of the GNU General Public License
 % along with ONSAS.  If not, see <https://www.gnu.org/licenses/>.
  
-function [deltaured, nextLoadFactorVals ] = computeDeltaU ( systemDeltauMatrix, systemDeltauRHS, dispIter, redConvDeltau, analysisSettings, nextLoadFactorVals, currDeltau  )
+function [deltaured, nextLoadFactorVals ] = computeDeltaU ( systemDeltauMatrix, systemDeltauRHS, dispIter, redConvDeltau, analysisSettings, nextLoadFactorVals, currDeltau, timeIndex  )
 
   convDeltau = redConvDeltau ;
 
   if strcmp( analysisSettings.methodName, 'arcLength' )
 
     aux = systemDeltauMatrix \ systemDeltauRHS ;
-
-    incremArcLen = analysisSettings.incremArcLen ;
-
+		
+			if length(analysisSettings.incremArcLen) > 1
```

Comment:
I suggest fixing the indentation in these lines

Reply:
done in [eb890ee](https://github.com/ONSAS/ONSAS.m/pull/539/commits/eb890ee6649551252ba7eb64341964262e81dc31)